app/controllers/admin/works_controller.rb

```diff
@@ -392,4 +398,22 @@ def cancel_url
       admin_works_path
     end
     helper_method :cancel_url
+
+    # If this work is an oral history,
+    # return true if the user needs to (re)calculate combined audio derivatives.
+    def need_combined_audio_derivatives?
+      return false
```

Comment:
This seems to be a mistake, you immediately return false without checking any conditions... which means this method will always return false. Which makes me suspect any tests that are present here must not be suffiiciently reliable, if they pass despite this broken method?

I may be wrong?

Reply:
Ohhh no, you're not wrong. I'm wrong. Will fix. And add tests.